components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -4394,7 +4394,13 @@ document.observe("xwiki:dom:updated", init);
                   );
           json.def_translated &amp;&amp; c.insert(elt('p').update(json.def_translated.replace(/\s*\n\s*/, ' ')));
 
-          var labels = {'alternative_term' : "$services.localization.render('phenotips.widgets.helpButtons.phenotype.synonym')", 'parents' : "$services.localization.render('phenotips.widgets.helpButtons.phenotype.typeOf')"};
+          var labels = {'synonym' : "$escapetool.javascript($services.localization.render('phenotips.widgets.helpButtons.phenotype.synonym'))",
+            'parents' : "$escapetool.javascript($services.localization.render('phenotips.widgets.helpButtons.phenotype.typeOf'))",
```

Comment:
Should be "phenotips.widgets.helpButtons.**ordoDisease**.parents", which needs to be added in the translations file.